draft-link-v6ops-464xlaton.xml

```diff
@@ -171,14 +171,71 @@ CLAT Addresses Considerations
+</t>
+<t>
+In accordance with Section 5.4 of <xref target="RFC4862"/>, the node MUST perform Duplicate Address Detection for its dedicated CLAT address.
+</t>
+<t>
+If the dedicated CLAT address is obtained via SLAAC, the node SHOULD ensure that the address is checksum-neutral for the given local IPv4 CLAT address and the NAT64 prefix (this means that the local IPv4 address needs to be assigned/known before the IPv6 one is configured).
+Using a checksum-neutral CLAT address provides the following benefits:
+</t>
+<ul>
+<li>
+<t>Better performance as CLAT doesn't need to reclaclulate the checksum.</t>
+</li>
+<li>
+<t>
+If a protocol uses the standard IP checksum, CLAT doesn't need to recalculate the checksum.
+That improves the chances of the protocol to work via CLAT even if CLAT is not aware of the protocol semantic.
+</t>
+</li>
+</ul>
 
+<t>
+The node SHOULD generate the different interface id for the CLAT address when connecting to the different networks, even if the NAT64 prefix and the local IPv4 CLAT address do not change. 
+To achieve that, the node SHOULD generate a random checksum-neutral CLAT address every time.
 </t>
 </section>
 
 <section anchor="v4addr">
 <name>
 CLAT IPv4 Addresses
 </name>
+<t>
+There are two different cases of IPv4 usage in 464XLAT deployments:
+</t>
+<ul>
+<li>
+<t>
+Wireline network archutecture, as defined in Section 4.1 of <xref target="RFC6877"/>. In that case the node performing CLAT functions also extends the network downstream and provides network connectivity services to other connected systems.
+Those systems can be physical (e.g. various clients connected to a CPE router), or logical (e.g. virtual systems running on a node, while the host system acts as a router and performs CLAT). 
+In all those cases, systems behind the CLAT node usually use <xref target="RFC1918"/> addresses.
+</t>
+</li>
+<li>
+<t>
+Wireless network archutecture, as defined in Section 4.2 of <xref target="RFC6877"/>.
+In that case the CLAT node provides IPv4 address and the default route to the local node's network stack only.
+It should be noted that <xref target="RFC6877"/> implies that the second deployment scenario is limited to 3GPP cases, while currently it is also deployed in other types of networks, such as enterprise networks and public WiFis, where hosts (as mobile phones, laptops and desktops) use CLAT to provide connectivity to IPv4-only local applications.
+</t>
+</li>
+</ul>
+<t>
+In the latter case, the CLAT needs a single IPv4 address only. The node providing CLAT functions to local applications SHOULD use IPv4 addresses from the dedicated 192.0.0.0/29 range (<xref target="RFC7335"/>), reserved for IPv4 continuity solutions, including, but not limited to 464XLAT.
+If the node runs multiple CLAT instances (see <xref target="multihomed"/>), the node SHOULD use different local IPv4 addresses for each CLAT instance.
+It means that it might it be possible to run more that 8 CLAT instances per node.
```

Comment:
8, or 6? Are we reusing the first and last address from this range as just another source address? See https://www.rfc-editor.org/rfc/rfc6333#section-10

Reply:
In case of CLAT those addresses  are internal to the host and never appears on the wire, so .0, .1 and .7 can be used. MacOS, for example, configures that address with 255.255.255.255 mask:
inet 192.0.0.2 netmask 0xffffffff broadcast 192.0.0.2
As per https://www.rfc-editor.org/rfc/rfc7335.html#section-4

"A host MUST NOT
   enable two active IPv4 continuity solutions simultaneously in a way
   that would cause a node to have overlapping 192.0.0.0/29 address
   space."

Therefore .1 can't be used as DS-lite router if 464xlat is present.

Let me add some text (in another PR) about that.